src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/PerfTests/DebuggerSteppingTestcase.xml

```diff
@@ -51,7 +49,6 @@
         <StepOver waitForBreakOrEnd="true"/>
       </Debug>
       <WaitForCodeMarkers />
-      <MeasureTimeStop /><!-- End timer (Not sure if correct command) -->
       <Debug>
         <StopDebugging />
```

Comment:
add the check that verify wrong dlls being not loaded.

      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoCSharp" />
      <VerifyRoslynModulesLoadedStatus ExpectedStatus="Basic" />